File: Mikrostoritve_/Customer/Customer/src/main/java/com/example/Customer/Customer.java
package com.example.Customer;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
@Document(collection = "customers")
public class Customer {
    @Id
    public String id;
    public String nameAndSurname;
    public String dateOfBirth;
    public String gender;
    public String email;
    public String telephone;
    public String address;
    public String typeOfSubscription;
    public String dateOfSubscription;
    public String paymentMethod;
    public String validTill;
    public  String status;

}

File: Mikrostoritev2/src/main/kotlin/com/mikrostoritev2user/quarkus/grpc/Customer.kt
package com.mikrostoritev2user.quarkus.grpc

    data class Customer(
    val nameAndSurname: String,
    val dateOfBirth: String,
    val gender: String,
    val email: String,
    val telephone: String,
    val address: String,
    val note: String,
    val dateInserted: String,
    val dateUpdated: String,
    val customerId: String
    )